src/main/java/io/codelex/polymorphism/practice/exercise6/MainTest.java

```diff
@@ -0,0 +1,125 @@
+package io.codelex.polymorphism.practice.exercise6;
+
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.Scanner;
+
+public class MainTest {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        scanner.useLocale(Locale.US);
+        DecimalFormat format = new DecimalFormat("0.#");
+
+        List<String> animals = new ArrayList<>();
+
+        Vegetable vegetable = new Vegetable(0);
+        Meat meat = new Meat(0);
+
+        Cat cat = new Cat(null, "Cat", 0, 0, null, null);
```

Comment:
no sākuma noteikti negribi jau visus objektus uztaisīt, to dari brīdi, kad ir skaidrs, kāds dzīvnieks vajadzīgs, šajā gadījumā iekš switch

Reply:
Darīts, paldies!